ui/app/utils/classes/abstract-stats-tracker.js

```diff
@@ -36,18 +64,18 @@ export default Mixin.create({
       assert('Url must be defined', url);
 
       yield this.get('fetch')(url)
-        .then(res => res.json())
-        .then(frame => this.append(frame));
+        .then(jsonWithDefault({ error: true }))
+        .then(frame => this.handleResponse(frame));
     } catch (error) {
```

Comment:
Yep, I wrote out the rationale on [the PR that introduced stat trackers](https://github.com/hashicorp/nomad/pull/4635).

> This may sound a lot like what Ember Data does, but this isn't using Ember Data. This data isn't stateful (no writes), and it isn't necessarily globally persistent (created and deleted on demand). Although fetching and normalizing the data does overlap in behavior with adapters and serializers, given the nature of client and allocation stats, I think it makes more sense to keep this data out of the Ember Data store.

And yes, similar rationale for things like leader and regions. They're one-off calls that aren't going to change, have no relationships, have no computed properties, aren't CRUD, etc. If that changes some day, I'll convert them into ED models.